.azure-pipelines.yml

```diff
@@ -156,28 +156,30 @@ jobs:
       displayName: Set tag name
       condition: startsWith(variables['Build.SourceBranch'], 'refs/tags/')
     - bash: |
-        echo "##vso[task.setvariable variable=tagName;]master"
-      displayName: Set tag name to "master"
+        echo "##vso[task.setvariable variable=tagName;]dev"
+      displayName: Set tag name to "dev"
       condition: eq(variables['Build.SourceBranch'], 'refs/heads/master')
 
     - bash: |
         mkdir $BUILD_BINARIESDIRECTORY/bundle
         cp target/release/{wasmtime,wasm2obj}.exe $BUILD_BINARIESDIRECTORY/bundle
       displayName: Copy binaries (Win)
       condition: and(succeeded(), eq( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
+
+    - bash: |
+        set -e
+        export WT_VERSION=`cat Cargo.toml | sed -n 's/^version = "\([^"]*\)".*/\1/p'`
+        "$WIX/bin/candle" -arch x64 -out target/wasmtime.wixobj installer/msi/wasmtime.wxs
+        "$WIX/bin/light" -out $BUILD_ARTIFACTSTAGINGDIRECTORY/wasmtime-$TAGNAME-x86_64-windows.msi target/wasmtime.wixobj -ext WixUtilExtension
+      displayName: Create installer (Windows)
+      condition: and(succeeded(), eq( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
+
     - bash: |
         mkdir $BUILD_BINARIESDIRECTORY/bundle
         cp target/release/{wasmtime,wasm2obj} $BUILD_BINARIESDIRECTORY/bundle
       displayName: Copy binaries (*nix)
       condition: and(succeeded(), ne( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
-    - task: ArchiveFiles@2
-      inputs:
-        rootFolderOrFile: $(Build.BinariesDirectory)/bundle
-        includeRootFolder: false
-        archiveType: 'zip'
-        archiveFile: '$(Build.ArtifactStagingDirectory)/wasmtime-$(tagName)-x86_64-windows.zip'
-      displayName: Archive files (Win)
-      condition: and(succeeded(), eq( variables['Agent.OS'], 'Windows_NT' ), eq( variables['build_type'], 'release' ))
```

Comment:
I've found that for automated installation of tools it's sometimes nifty to still have a tarball-like installer for Windows that's not just the executable ones. Perhaps both an `*.msi` could be created as well as a `*.tar.xz` to match the other platforms.

(I'm assuming that some sort of installer website would smoth over "what to choose" here as well)

Reply:
Good point, yes. I left the zip file in place with the latest version here.